pages/docs/integrations/langchain/python.md

```diff
@@ -4,37 +4,44 @@ description: Langchain users can integrated with Langfuse in seconds using the i
 
 # Langchain integration (Python)
 
-Langfuse integrates with Langchain using the [Langchain Callbacks](https://python.langchain.com/docs/modules/callbacks/). Thereby, the Langfuse SDK automatically creates a nested trace for the abstractions offered by Langchain.
+Langfuse integrates with Langchain using [Langchain Callbacks](https://python.langchain.com/docs/modules/callbacks/). Thereby, the Langfuse SDK automatically creates a nested trace for the abstractions offered by Langchain.
 
-Add the handler as a callback when running your Langchain model/chain/agent:
+Simply add the Langfuse handler as a callback when running your Langchain model/chain/agent to start capturing traces from your executions:
 
-```python /callbacks=[handler]/
+```python /callbacks=[langfuse_handler]/
 # Initialize Langfuse handler
 from langfuse.callback import CallbackHandler
-handler = CallbackHandler(PUBLIC_KEY, SECRET_KEY)
+
+langfuse_handler = CallbackHandler(
+    public_key=LANGFUSE_PUBLIC_KEY, secret_key=LANGFUSE_SECRET_KEY
+)
 
 # Setup Langchain
 from langchain.chains import LLMChain
 ...
-chain = LLMChain(llm=llm, prompt=prompt, callbacks=[handler])
+chain = LLMChain(llm=llm, prompt=prompt, callbacks=[langfuse_handler])
 
 # Add Langfuse handler as callback
-chain.run(input="<user_input", callbacks=[handler])
+chain.run(input="<user_input>", callbacks=[langfuse_handler])
 ```
 
 Langchain expression language (LCEL)
-```python /config={"callbacks":[handler]}/
-chain.invoke(input, config={"callbacks":[handler]})
+
+```python /config={"callbacks":[langfuse_handler]}/
```

Comment:
you did not break it but the formatting does not work. We should adjust this when re-working the references.